theme/partials/awards.hbs

```diff
@@ -5,44 +5,45 @@
   </header>
   <section id="awards">
     {{#each resume.awards}}
-    <header class="clear">
-      <span class="space-left">
-      {{#if @first}}
-      <span class="fa fa-caret-down"></span>
-      {{else}}
-      <span class="fa fa-caret-right"></span>
+    <section class="award-item">
+      {{#if summary}}
+        {{#if @first}}
```

Comment:
You could create a partial called for example "toggleItem" that accepts "id" and "checked(boolean)" as arguments and renders this input tag